README.md

```diff
@@ -1,26 +1,64 @@
 # Golden Sample Angular App
 
-This example project is a reference implementation to showcase a number of best practices to use when building a new Angular SPA that leverages Backbase components and libraries. The project is regularly updated, so come back often to check for new improvements. For the best experience, these VSCode extenstions for Nx and Jest should be installed. Jest runner allows you to easly run isolated test as you are developing. Nx Console allows you to easily find and run all the possible Nx Commands.
+The mission of the golden sample is to provide examples of code structure, configuration, and best practices we would like to instill during journey development that would be beneficial as a reference to the audience.
+
+
+## Table of Contents
+* [What is app for](#what-is-app-for)
+* [Requirements](#requirements)
+* [Authentication details](#authentication-details)
+* [Generate an application](#generate-an-application)
+* [Generate a library](#generate-a-library)
+* [Development server](#development-server)
+* [Code scaffolding](#code-scaffolding)
+* [Build](#build)
+* [Tests](#tests)
+* [Understand your workspace](#understand-your-workspace)
+* [Running with docker](#running-with-docker)
+* [Package as a runnable Docker container](#package-as-a-runnable-docker-container)
+* [Further help](#further-help)
+
+
+
+
+
+## What is app for
+- This example project is a reference implementation to showcase a number of best practices to use when building a new Angular SPA that leverages Backbase components and libraries. The project is regularly updated, so come back often to check for new improvements.
+- This prototype covers:
+    - Authentication in the app
+    - Enable support of the multiple languages
+    - Integration with a theme
+    - Using existing journey installed from 
+    - local NPM repository
+    - Configuration of the journeys
+    - Configuration of the entitlements
+    - Communication between journeys
```

Comment:
Could we add links to the community for some of this points? Or it is a part of a different task that we planned?

Reply:
Yes, this one is for another ticket and we will definitely work on it soon.